tools/src/h5perf/sio_engine.c

```diff
@@ -1266,13 +1266,6 @@ do_fclose(iotype iot, file_descr *fd /*out*/)
  * Programmer:  Albert Cheng 2001/12/12
  * Modifications: Support for file drivers. Christian Chilan, April, 2008
  */
-/* Disable warning for "format not a string literal" here -QAK */
-/*
- *      This pragma only needs to surround the snprintf() calls with
- *      'temp' in the code below, but early (4.4.7, at least) gcc only
- *      allows diagnostic pragmas to be toggled outside of functions.
- */
-H5_GCC_DIAG_OFF("format-nonliteral")
```

Comment:
There is a matching 'H5_GCC_DIAG_ON("format-nonliteral")' in line 1331.  Should they both be kept or removed?

Reply:
The other line should be removed

Reply:
Done.